Documentation/how-to-run-examples.md

```diff
@@ -2,6 +2,285 @@
+$ sudo apt-get install nnstreamer
+```
+Note that this may install tensorflow packaged by us.
+
+* Download nnstreamer example :
+```
+$ sudo add-apt-repository ppa:nnstreamer-example/ppa
+$ sudo apt-get update
+$ sudo apt-get install nnstreamer-example
+$ cd /usr/lib/nnstreamer/bin
+```
+
+As of 2018/10/13, we support 16.04 and 18.04
+
+## Build examples (Ubuntu 16.04 and 18.04)
+* https://github.com/nnstreamer/nnstreamer/blob/master/Documentation/getting-started.md
+
+- Install related packages before building nnstreamer and examples.
+1. ninja-build, meson (>=0.42)
+2. liborc (>=0.4.25, optional)
+3. tensorflow, protobuf (>=3.4.0)
+4. tensorflow-lite
+
+- Build options (meson)
+  * You can find the option definitions here: [meson options](https://github.com/nnstreamer/nnstreamer/blob/main/meson_options.txt)
+  * For more information about meson build options itself, see [here](https://mesonbuild.com/Build-options.html).
+
+  For example, to build and install NNStreamer by enabling C-API and disabling tensorflow2-lite,
+   ```
+   $ meson --prefix=${NNST_ROOT} --sysconfdir=${NNST_ROOT} --libdir=lib --bindir=bin --includedir=include -Denable-capi=true -Dtflite2-support=disabled build
+   ```
+
+- Build source code
+```
+# Install packages for python example
+$ sudo apt-get install python-gi python3-gi
+$ sudo apt-get install python-gst-1.0 python3-gst-1.0
+$ sudo apt-get install python-gst-1.0-dbg python3-gst-1.0-dbg
+
+# Set your own path to install libraries and header files
+$ sudo vi ~/.bashrc
+
+export NNST_ROOT=$HOME/nnstreamer
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$NNST_ROOT/lib
+export GST_PLUGIN_PATH=$GST_PLUGIN_PATH:$NNST_ROOT/lib/gstreamer-1.0
+# Include NNStreamer headers and libraries
+export C_INCLUDE_PATH=$C_INCLUDE_PATH:$NNST_ROOT/include
+export CPLUS_INCLUDE_PATH=$CPLUS_INCLUDE_PATH:$NNST_ROOT/include
+export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:$NNST_ROOT/lib/pkgconfig
+
+$ source ~/.bashrc
+
+# Download source, then compile it.
+# Build and install nnstreamer
+$ git clone https://github.com/nnstreamer/nnstreamer.git nnstreamer.git
+$ cd nnstreamer.git
+$ meson --prefix=${NNST_ROOT} --sysconfdir=${NNST_ROOT} --libdir=lib --bindir=bin --includedir=include build
+$ ninja -C build install
+$ cd ..
+
+# Build and install examples
+$ git clone https://github.com/nnstreamer/nnstreamer-example.git nnstreamer-example.git
+$ cd nnstreamer-example.git
+$ meson --prefix=${NNST_ROOT} --libdir=lib --bindir=bin --includedir=include build
+$ ninja -C build install
+$ rm -rf build
+$ cd ..
+```
+
+- Download tensorflow-lite model & labels.
+```
+$ cd $NNST_ROOT/bin/
+$ mkdir tflite_model       # for the example image classification
+$ cd tflite_model
+$ wget https://github.com/nnsuite/testcases/raw/master/DeepLearningModels/tensorflow-lite/Mobilenet_v1_1.0_224_quant/mobilenet_v1_1.0_224_quant.tflite
+$ wget https://github.com/nnsuite/testcases/raw/master/DeepLearningModels/tensorflow-lite/Mobilenet_v1_1.0_224_quant/labels.txt
+$ cd ..
```

Comment:
plz check example repo, tflite-object detection example.

remove line 106 ~ 108
now tflite_model directory is default.

Reply:
It is better to update this guide to use 'get-model.sh' in example repo.

Reply:
Changed to use get-mode.sh. Thanks.